WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -217,6 +217,7 @@ class MySiteViewModel @Inject constructor(
     val onShare = _onShare
     val onTrackWithTabSource = _onTrackWithTabSource as LiveData<Event<MySiteTrackWithTabSource>>
     val selectTab: LiveData<Event<TabNavigation>> = _selectTab
+    private var shouldMarkUpdateSiteTitleTaskComplete = false
```

Comment:
I'm not happy about this, but I didn't want to pass this boolean through the dialog code :man_shrugging: . Would be happy if there is a alternative.

Reply:
I couldn't find a better way for this either. Since there two more `var`s in this view model I guess it might be ok to add this 🤔